ui/src/kapacitor/components/config/SlackConfig.tsx

```diff
@@ -183,29 +214,31 @@ class SlackConfig extends PureComponent<Props, State> {
   }
 
   private handleSubmit = async e => {
-    const {isNewConfig} = this.props
     e.preventDefault()
+
+    const {workspace} = this.state
```

Comment:
could you share your thoughts on why this would be from state as opposed to other state destructuring that has been modularized into getters?

Reply:
State is a little different to me than props, since there is no computed value needed I felt fine about destructuring from state.